src/moepkg/commandlineutils.nim

```diff
@@ -45,6 +46,7 @@ const exCommandList: array[65, tuple[command, description: string]] = [
   (command: "log", description: "                  | Open the log viewer"),
   (command: "ls", description: "                   | Show the all buffer"),
   (command: "lsw", description: "                  | Show the all workspace"),
+  (command: "man", description: "                  | Show the given UNIX manual page, if available"),
```

Comment:
@fox0430

`man` will write an error message to the terminal if the queried page does not exist.  The user shall be responsible for calling only existing manual pages.